modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path

```diff
@@ -509,6 +509,27 @@
 	<td>//div/label[@class='control-label' and contains(.,'${key_customFieldName}')]/..//p</td>
 	<td></td>
 </tr>
+<!--SUBSCRIPTIONS-->
+<tr>
+	<td>CLICK_SUBSCRIPTIONS</td>
+	<td>//a[contains(text(),'Subscriptions')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CLICK_OVERRIDE</td>
+	<td>//span[contains(text(),'Override Subscription Settings')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PAYMENT_SUBSCRIPTIONS</td>
+	<td>//h4[contains(text(), 'Payment Subscription')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DELIVERY_SUBSCRIPTIONS</td>
+	<td>//h4[contains(text(), 'Delivery Subscription')]</td>
+	<td></td>
+</tr>
```

Comment:
Please create a single path with a variable so that it can be used for both cases